src/components/Composer/index.js

```diff
@@ -376,12 +364,19 @@ class Composer extends React.Component {
             + parseInt(computedStyle.paddingTop, 10);
             const numberOfLines = getNumberOfLines(this.props.maxLines, lineHeight, paddingTopAndBottom, this.textInput.scrollHeight);
             updateIsFullComposerAvailable(this.props, numberOfLines);
-            this.setState({
-                numberOfLines,
-            });
+            this.setState({numberOfLines});
         });
     }
 
+    focus(onDone, delay) {
```

Comment:
Same missing JSDocs

Reply:
Also, why do we need `delay` here?

Reply:
The delay is also needed on the web to re-focus the text input correctly